sources/datasource/kubelet.go

```diff
@@ -176,11 +185,23 @@ func (self *kubeletSource) getAllContainers(url string, start, end time.Time) ([
 	}
 
 	// TODO(vmarmol): Use this for all stats gathering.
-	// Don't include the Kubernetes containers, those are included elsewhere.
+	// Only include the Kubernetes infra containers, the others are collected via the kubelet pod stats API.
```

Comment:
Sorry that's a *bad* comment! Will update - raw containers are of course retained.